### DISCUSSION TITLE
Interaction with the camera

### DISCUSSION THREAD

**Discussion by torannn** [jeertmans/manim-slides#139] [Q&A] [answered]
Can we develop the interactivity within manim-slides? Juts like some basic interaction like manoeuvring with the camera?

**Comment by jeertmans** [comment, marked as answer]
I'm afraid it is not possible, and it will hardly be. The main issue it that maneuvering the camera means that we need to do real time rendering, and Manim Slides does not do any rendering.

If Manim Slides were to use manim for rendering, I'm not even sure that it will be fast enough to render in real-time :-/

I know 3b1b developed an interactive mode for ManimGL, but I have never tested it.
